README.md

```diff
@@ -1,21 +1,595 @@
 # GMRIGHT2-coll
```

Comment:
Logo name

Reply:
Gmright2-coll is the logo name in each application communication and all the data source label